packages/blade/.storybook/react/preview.js

```diff
@@ -43,12 +43,50 @@ export const parameters = {
           'makeTypographySize',
           'useTheme',
           'overrideTheme',
+          'Accessibility',
         ],
         'Components',
+        [
+          'Typography',
+          'Amount',
+          'Icons',
+          'Button',
+          'IconButton',
+          'Link',
+          // Input group
+          'Input',
+          'Dropdown',
+          'Checkbox',
+          'Radio & RadioGroup',
+          'Switch',
+          // Layout and complex interactive elements group
+          'Layout Primitives (Box)',
+          'Card',
+          'Modal',
+          'BottomSheet',
+          'Divider',
+          'List',
+          'Accordion',
+          'Collapsible',
+          // feedback group
+          'Indicator',
+          'Badge',
+          'Counter',
```

Comment:
This sorting makes sense here because it has this `// feedback group`, `// Layout and complex interactive elements` headings.

But when I see it in storybook, it feels a bit random. The consumers won't understand this grouping because they won't have the detailed context no? The alphabetical order felt a bit simpler to read 🙈 

<img width="232" alt="image" src="https://github.com/razorpay/blade/assets/30949385/ff2b569f-b2fd-4d1b-af6b-2414e0dc644e">

Reply:
Ideally there should've been option to put more subheadings in storybook 🙈

Reply:
we can put more subheadings we can create groups as well. I thought to do it but then i can't find components from top level. should we do that?

Reply:
They end up like directory no? or will it look like proper heading? and will it stay expanded 🤔 ?

Reply:
If I put `/` then it goes as a subdirectory eg: `Components/Feedback` but I can do `Components-Feedback` like that

Reply:
Oh that would be great. I tried putting `-` locally but didn't seem to work 🤔 might be missing somehing

Reply:
looks like this. honestly doesn't look very intuitive 🙈 
<img width="234" alt="Screenshot 2023-08-03 at 1 06 57 PM" src="https://github.com/razorpay/blade/assets/11384858/1c8f37c2-c658-44e9-a815-bbb7a19cd071">

the other is this option with `/` which creates sub directories. This still looks better but then discoverability becomes a challenge.yes people should use `search` but you know what i mean here 😆 
<img width="228" alt="Screenshot 2023-08-03 at 1 08 38 PM" src="https://github.com/razorpay/blade/assets/11384858/9425da30-6b59-4b5a-bc26-298bb9d72783">

Reply:
@saurabhdaware 👆

Reply:
Should we let it be alphabetic order then? Not sure if they will be familiar with the terminology of Feedback group etc. They will see Alert on design and will come to our docs to search "Alert" itself 🤔

Reply:
yes isliye i thought to group it internally and not confuse consumers but when you start looking for components you'll find related things nearby